html/semantics/scripting-1/the-script-element/nomodule-set-dynamically-on-external-classic-scripts.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Dynamically inserted classic script with noModule set to false must not load</title>
+<link rel="author" title="Yusuke Suzuki" href="mailto:utatane.tea@gmail.com">
+<link rel="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<!-- Load this script synchronously to ensure test cases below can load it in 200ms -->
+<script src="resources/set-window-executed.js"></script>
```

Comment:
All of these should be `set-script-executed.js` (or rename the file to `set-window-executed.js`)